temp/static_analysis.md

```diff
@@ -0,0 +1,549 @@
+```pycon
+>>> stress_S1 = static_simulation.nodal_principal_stress(
+...     components=[1], named_selection=named_selections[0]
+... )
+```
+
+### Extract elemental Von Mises stress everywhere at all sets
+```pycon
+>>> stress_VM = static_simulation.elemental_von_mises_eqv_stress()
+```
+
+### Extract equivalent elemental nodal elastic strain for a selection at set 1
+```pycon
+>>> elastic_strain_XY = static_simulation.nodal_elastic_strain(
+...     components=["XY"], selection=selection, set_ids=[1]
+... )
+```
+
+### Extract first principal nodal strain for a selection at set 1
+```pycon
+>>> elastic_strain_E1 = static_simulation.nodal_elastic_principal_strain(
+...     components=[1], selection=selection, set_ids=[1]
+... )
+```
+Write in the doc that this is E1 so that researching in doc will get you there.
+
+### Extract nodal plastic strain for a selection at set 1
+```pycon
+>>> plastic_strain = static_simulation.nodal_plastic_strain(selection=selection, set_ids=[1])
+```
+
+## Manipulate results with DataObject
+
+### Explore a DataObject
+Print a DataObject
+
+```pycon
+>>> print(stress_S1)
+DPF DataObject:
+Nodal stress S1
+    on:
+    - named selection {named_selection_name} # if present
+    - N nodes/elements/faces:
+        [1, ..., 45]  # summarized list representation if less than X characters long
+    - N steps/frequencies/times:
+        [...]
+    with:
+    - S1 stress (Pa)
+        [range]
+```
+Or print-out the DataObject in a dataframe style,
+with an index on the main supporting mesh entity id
+```pycon
+DPF DataObject:
+Nodal stress S1 [on *named_selection_name*] # if present
+ node ID   step  component   value
+       1      1         S1     0.1
+       2      1         S1     0.2
+       3      1         S1     0.3
+       ...
+       1      2         S1     0.2
+       2      2         S1     0.3
+       3      2         S1     NaN
+```
+
+Get the name of the DataObject
+print(stress_S1.name)
+Nodal stress S1
+
+Get the mesh support of the DataObject
+print(stress_S1.mesh)
+
+Get the shape of the DataObject
+```pycon
+>>> print(stress_S1.shape)
+list of lengths of each dimension present: [nb_steps, nb_entities,
+```
+
+Get specific data from a DataObject (as a new DataObject)
+```pycon
+>>> stress_S1_step_1 = stress_S1[1]  # Equivalent to stress_S1[0, ...] or stress_S1[0, :, :]
+```
+or
+```pycon
+>>> stress_S1_step_1 = stress_S1.step(1)
+```
+
+```pycon
+>>> print(stress_S1_step_1)
+Nodal stress S1
+For:
+- steps: 1
+- named selection: {named_selection[0]}
+```
+
+### Get the minimum, maximum, mean of the DataObject 
+Use DPF operators for efficiency.
+
+Across all dimensions present in the specific DataObject, unless specified as argument
+Take inspiration from the "axis" argument for ``numpy.ndarray.amax``.
+Do we want a ``nanmax``? Which propagates NaN values? (Whether we have actual NaNs or not)
+These would return DataObjects or a scalar.
```

Comment:
```suggestion
Do we want a ``nanmax``? Which propagates NaN values? (Whether we have actual NaNs or not)
These would return DataObjects or a scalar.
```
I don't think we need that... if you return the data as an array, you can filter it.